packages/documentation/src/stories/foundation/color/color.module.scss

```
@@ -1,5 +1,25 @@
+@use 'sass:map';
+@use 'sass:list';
 @use '@swisspost/design-system-styles/core' as post;
 
+$all_exported_colors: () !default;
+$all_exported_colors: map-merge(
```

```suggestion
$all_exported_colors: map.merge(
```
Please update all the other occurences as well.